analytics.js

```diff
@@ -1,53 +1,66 @@
 'use strict';
 
+let keenClient = null;
+
 /**
  * Send a custom analytics event to Google & Optimizely.
  *
  * @param category
  * @param action
  * @param label
  */
-function analyze(category, action, label) {
-    var stringified = [category, action, label]
-      .filter(function(n) { return n; })
-      .join(':');
+function analyze(identifier, data) {
+  const identifiers = identifier.split(':');
 
-    if (process.env.NODE_ENV !== 'production') {
-        console.info('Sent an analytics event - ' + stringified)
-    }
+  const category = identifiers[0];
+  const action = identifiers[1];
+  const label = identifiers[2];
```

Comment:
I think we're okay since `identifiers[2]` will just return `undefined` if there isn't enough "partitions" in the string. Since these are only used for the `ga()` call below, I'd even consider just inlining them into the function args.